lib/agent.js

```diff
@@ -136,7 +136,9 @@ Agent.prototype.start = function (opts) {
     this._conf.active = false
     return this
   } else if (this._conf.logLevel === 'trace') {
-    var _ancestors = ancestors(module)
+    var _ancestors = ancestors(module).filter(Boolean)
+    var userModule = process.argv[1]
+    _ancestors.push(userModule)
     var basedir = path.dirname(_ancestors[_ancestors.length - 1])
```

Comment:
Seems to me like `_ancestors` should be left as-is and, instead, the `basedir` should just be set to `path.dirname(process.argv[1])` rather than pulling off the ancestors list. What do you think, @watson?

Reply:
Interesting approach. I hadn't considered that. Lemme just check some edge cases...

Reply:
That all seems fine. I don't know why I didn't think of that approach previously 😄

Reply:
Great. I'll re-implement to include @Qard's suggestion and set `basedir` to be `path.dirname(process.argv[1])`  

There does seem to be an issue with how ancestors loops through the module. It always pulls `null` into the array, which is why I filtered out the value. But that's perhaps an issue with `ancestors` or the `module` itself?